.github/workflows/lint.yml

```diff
@@ -4,15 +4,17 @@ on:
 
 jobs:
   ament_lint:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
+    container:
+      image: rostooling/setup-ros-docker:ubuntu-bionic-ros-eloquent-ros-base-latest
```

Comment:
For master branch, do we need a "foxy" or "master" version of this base image?

Reply:
The lint workflow (even before this commit) is relying on prebuilt binaries from APT packages, so we don't have a way to test master. As far as Foxy is concerned, yes, we should mass-update to Focal/Foxy once we have stable APT packages.